### ISSUE TITLE
Fargate doesn't support docker shm parameter needed for stable Firefox and Chrome

### ISSUE THREAD

**Issue by simon-buxton-kg** [xilution/xilution-selenium-grid#8]
In order to avoid random crashes it seems the Firefox and Chrome images need increased shared memory when running in a container, as per: 

> When executing docker run for an image with Chrome or Firefox please either mount -v /dev/shm:/dev/shm or use the flag --shm-size=2g to use the host's shared memory.

from https://github.com/SeleniumHQ/docker-selenium#running-the-images

However at the moment AWS Fargate doesn't support this:

>sharedMemorySize
The value for the size (in MiB) of the /dev/shm volume. This parameter maps to the --shm-size option to docker run.
Note
If you are using tasks that use the Fargate launch type, the sharedMemorySize parameter is not supported.
Type: Integer 

from https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_linuxparameters

Just wondering if anyone is actually seeing this problem when trying to use Fargate to run Selenium nodes?

**Comment by simon-buxton-kg** [comment]
These Firefox and Chrome bugs go into more detail about the problem:

https://bugs.chromium.org/p/chromium/issues/detail?id=519952

https://bugzilla.mozilla.org/show_bug.cgi?id=1338771#c10

**Comment by alovak** [comment]
You can disable using /dev/shm by passing --disable-dev-shm-usage option when you run Chrome.

https://bugs.chromium.org/p/chromium/issues/detail?id=736452

**Comment by blopker** [comment]
Is there an equivalent for Firefox?

**Comment by taktakpeops** [comment]
No, Firefox doesn't offer such a flag based on their documentation: https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options#How_to_use_command_options

**Comment by harakiro** [comment]
> --disable-dev-shm-usage

Fixed it for me! Thank you!
